src/main/java/org/folio/circulation/resources/renewal/RenewalResource.java

```diff
@@ -172,13 +175,15 @@ private void renew(RoutingContext routingContext) {
 
     findLoan(bodyAsJson, loanRepository, itemRepository, userRepository, errorHandler)
       .thenApply(r -> r.map(loan -> RenewalContext.create(loan, bodyAsJson, webContext.getUserId())))
-      .thenComposeAsync(r-> refuseWhenPatronIsInactive(r, errorHandler, USER_IS_INACTIVE))
+      .thenComposeAsync(r -> refuseWhenPatronIsInactive(r, errorHandler, USER_IS_INACTIVE))
       .thenComposeAsync(r -> refuseWhenRenewalActionIsBlockedForPatron(
         manualPatronBlocksValidator, r, errorHandler, USER_IS_BLOCKED_MANUALLY))
       .thenComposeAsync(r -> refuseWhenRenewalActionIsBlockedForPatron(
         automatedPatronBlocksValidator, r, errorHandler, USER_IS_BLOCKED_AUTOMATICALLY))
       .thenComposeAsync(r -> refuseIfNoPermissionsForRenewalOverride(
         overrideRenewValidator, r, errorHandler))
+      .thenCompose(r -> r.after(ctx -> lookupOverdueFinePolicy(ctx, overdueFinePolicyRepository, errorHandler)))
```

Comment:
I believe this is also part of another PR - https://github.com/folio-org/mod-circulation/pull/1412/files
Does it make sense to finish and merge that one first? That will make this PR slightly shorter and also will help to avoid the need to resolve conflicts after every change.

Reply:
Yes, that makes sense (and was the plan actually)

Reply:
Right, it is. #1412 seems to have all issues resolved by now so I can go ahead and merge that before too long.

Reply:
#1412 merged/closed and #1403 (this one) synced from master